### PR TITLE
added links of previous year team ans then created page for team 2021-22

### PR DESCRIPTION
**Pull request by amdpt323**

### CODE REVIEW

**Comment by abhishekshree** [comment]
The links don't look aesthetic as of now; make them look similar to the ones on the `students` page (for resources).

The folder stricture should be `student-team/2021-22.tsx` to get a route like `/student-team/2021-22`

File: pages/student-team.tsx [hunk 1]
```diff
@@ -224,6 +228,37 @@ export default function StudentTeam() {
         </CustomAutofitGrid4>
       </Container>
       <br />
+      <SectionTitle>Previous Year Web Teams</SectionTitle>
+      <Container>
```
Comment (abhishekshree) [inline pages/student-team.tsx:232]: Change link to ones on the students page

**Review by abhishekshree — CHANGES_REQUESTED** [review]
Put it in the student-team folder, rename it to just the academic year.

**Comment by 1-Harshit** [comment]
- for #6

**Comment by 1-Harshit** [comment]
@amdpt323 
Fix the the font size
image

**Comment by 1-Harshit** [comment]
- duplicate of #34

**Pull request closed without merging**